# Yuri Chornoivan <yurchor@ukr.net>, 2015. #zanata
# Yuri Chornoivan <yurchor@ukr.net>, 2016. #zanata
# Yuri Chornoivan <yurchor@ukr.net>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-08-29 12:24+0200\n"
"PO-Revision-Date: 2017-08-30 02:28+0000\n"
"Last-Translator: Yuri Chornoivan <yurchor@ukr.net>\n"
"Language-Team: Ukrainian <kde-i18n-uk@kde.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: uk\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2)\n"
"X-Generator: Zanata 4.6.2\n"

#: ../plugins/reposync.orig.py:42 ../plugins/reposync.py:42
msgid "download all packages from remote repo"
msgstr "отримати усі пакунки із віддаленого сховища"

#: ../plugins/reposync.orig.py:48 ../plugins/reposync.py:48
msgid "download only packages for this ARCH"
msgstr ""

#: ../plugins/reposync.orig.py:50 ../plugins/reposync.py:50
msgid "delete local packages no longer present in repository"
msgstr ""

#: ../plugins/reposync.orig.py:52 ../plugins/reposync.py:52
msgid "also download comps.xml"
msgstr ""

#: ../plugins/reposync.orig.py:54 ../plugins/reposync.py:54
msgid "download only newest packages per-repo"
msgstr ""

#: ../plugins/reposync.orig.py:56 ../plugins/reposync.py:56
msgid "where to store downloaded repositories "
msgstr "місце, де слід зберігати отримані сховища "

#: ../plugins/reposync.orig.py:58 ../plugins/reposync.py:58
msgid "operate on source packages"
msgstr ""

#: ../plugins/reposync.orig.py:98 ../plugins/reposync.py:98
#, python-format
msgid "[DELETED] %s"
msgstr ""

#: ../plugins/reposync.orig.py:100 ../plugins/reposync.py:100
#, python-format
msgid "failed to delete file %s"
msgstr ""

#: ../plugins/reposync.orig.py:110 ../plugins/reposync.py:110
#, python-format
msgid "Could not make repository directory: %s"
msgstr ""

#: ../plugins/reposync.orig.py:114 ../plugins/reposync.py:114
#, python-format
msgid "comps.xml for repository %s saved"
msgstr ""

#: ../plugins/debuginfo-install.py:58
msgid "install debuginfo packages"
msgstr "встановити пакунки debuginfo"

#: ../plugins/debuginfo-install.py:85
#, python-format
msgid "No match for argument: %s"
msgstr ""

#: ../plugins/debuginfo-install.py:92
msgid "Unable to find a match"
msgstr ""

#: ../plugins/migrate.py:45
msgid "migrate yum's history, group and yumdb data to dnf"
msgstr "перенести журнал yum, дані щодо груп та yumdb до dnf"

#: ../plugins/migrate.py:54
msgid "Migrating history data..."
msgstr "Переносимо дані журналу…"

#: ../plugins/repograph.py:50
msgid "Output a full package dependency graph in dot format"
msgstr "Вивести повний граф залежностей пакунків у форматі dot"

#: ../plugins/repograph.py:110
#, python-format
msgid "Nothing provides: '%s'"
msgstr "Нічого не надає: «%s»"

#: ../plugins/versionlock.py:32
#, python-format
msgid "Unable to read version lock configuration: %s"
msgstr "Не вдалося прочитати налаштування блокування версії: %s"

#: ../plugins/versionlock.py:33
msgid "Locklist not set"
msgstr "Список блокування не встановлено"

#: ../plugins/versionlock.py:34
msgid "Adding versionlock on:"
msgstr "Додаємо блокування версії для:"

#: ../plugins/versionlock.py:35
msgid "Adding exclude on:"
msgstr "Додаємо виключення для:"

#: ../plugins/versionlock.py:36
msgid "Deleting versionlock for:"
msgstr "Вилучаємо блокування версії для:"

#: ../plugins/versionlock.py:37
msgid "No package found for:"
msgstr "Не знайдено пакунка для:"

#: ../plugins/versionlock.py:38
msgid "Excludes from versionlock plugin were not applied"
msgstr "Виключення з додатка versionlock не було застосовано"

#: ../plugins/versionlock.py:39
msgid "Versionlock plugin: number of lock rules from file \"{}\" applied: {}"
msgstr ""

#: ../plugins/versionlock.py:40
msgid "Versionlock plugin: number of exclude rules from file \"{}\" applied: {}"
msgstr ""

#: ../plugins/versionlock.py:41
msgid "Versionlock plugin: could not parse pattern:"
msgstr ""

#: ../plugins/versionlock.py:119
msgid "control package version locks"
msgstr "керування блокуванням версій пакунків"

#: ../plugins/builddep.py:68
msgid "[PACKAGE|PACKAGE.spec]"
msgstr "[ПАКУНОК|ПАКУНОК.spec]"

#: ../plugins/builddep.py:79
#, python-format
msgid "'%s' is not of the format 'MACRO EXPR'"
msgstr "«%s» не записано у форматі «МАКРОС ВИРАЗ»"

#: ../plugins/builddep.py:84
msgid "packages with builddeps to install"
msgstr "пакунки із залежностями для збирання, які встановлюватимуться"

#: ../plugins/builddep.py:87
msgid "define a macro for spec file parsing"
msgstr "визначити макрос для обробки файла специфікацій"

#: ../plugins/builddep.py:90
msgid "treat commandline arguments as spec files"
msgstr "вважати аргументи рядка команди назвами файлів spec"

#: ../plugins/builddep.py:92
msgid "treat commandline arguments as source rpm"
msgstr "вважати аргументи рядка команди назвами rpm із кодом пакунків"

#: ../plugins/builddep.py:138
msgid "Some packages could not be found."
msgstr "Не вдалося знайти деякі з пакунків."

#. No provides, no files
#: ../plugins/builddep.py:157
#, python-format
msgid "No matching package to install: '%s'"
msgstr "Немає відповідних пакунків для встановлення: «%s»"

#: ../plugins/builddep.py:174
#, python-format
msgid "Failed to open: '%s', not a valid source rpm file."
msgstr "Не вдалося відкрити «%s», це не файл rpm із кодом пакунка."

#: ../plugins/builddep.py:187 ../plugins/builddep.py:203
#: ../plugins/builddep.py:220
msgid "Not all dependencies satisfied"
msgstr "Задоволено не усі залежності"

#: ../plugins/builddep.py:194
#, python-format
msgid "Failed to open: '%s', not a valid spec file: %s"
msgstr "Не вдалося відкрити «%s» — файл не є коректним файлом spec: %s"

#: ../plugins/builddep.py:213
#, python-format
msgid "no package matched: %s"
msgstr "немає відповідного пакунка: %s"

#: ../plugins/config_manager.py:35
msgid "manage dnf configuration options and repositories"
msgstr "керування налаштуваннями та записами сховищ dnf"

#: ../plugins/config_manager.py:41
msgid "repo to modify"
msgstr "сховище для внесення змін"

#: ../plugins/config_manager.py:44
msgid "save the current options (useful with --setopt)"
msgstr "зберегти поточні параметри (корисний з --setopt)"

#: ../plugins/config_manager.py:47
msgid "enable the specified repos (automatically saves)"
msgstr "увімкнути вказані сховища (автоматичне збереження)"

#: ../plugins/config_manager.py:50
msgid "disable the specified repos (automatically saves)"
msgstr "вимкнути вказані сховища (автоматичне збереження)"

#: ../plugins/config_manager.py:53
msgid "add (and enable) the repo from the specified file or url"
msgstr ""
"додати (і увімкнути) сховище із вказаного файла або за вказаною адресою"

#: ../plugins/config_manager.py:56
msgid "print current configuration values to stdout"
msgstr "вивести значення поточних налаштувань до stdout"

#: ../plugins/config_manager.py:59
msgid "print variable values to stdout"
msgstr "вивести значення змінних до stdout"

#: ../plugins/config_manager.py:76
msgid "Error: Trying to enable and disable repos at the same time."
msgstr "Помилка: спроба одночасного вмикання і вимикання сховищ."

#: ../plugins/config_manager.py:79
msgid "Error: Trying to enable already enabled repos."
msgstr "Помилка: спроба увімкнути уже увімкнені сховища."

#: ../plugins/config_manager.py:118
#, python-format
msgid "No matching repo to modify: %s."
msgstr "Немає відповідного сховища для внесення змін: %s."

#: ../plugins/config_manager.py:149
#, python-format
msgid "Adding repo from: %s"
msgstr "Додаємо сховище з %s"

#: ../plugins/config_manager.py:173
msgid "Configuration of repo failed"
msgid_plural "Configuration of repos failed"
msgstr[0] "Помилка під час налаштовування сховища"
msgstr[1] "Помилка під час налаштовування сховищ"
msgstr[2] "Помилка під час налаштовування сховищ"

#: ../plugins/config_manager.py:183
#, python-format
msgid "Could not save repo to repofile %s: %s"
msgstr "Не вдалося зберегти дані сховища до файла сховищ %s: %s"

#: ../plugins/local.py:117
msgid "Unable to create a directory '{}' due to '{}'"
msgstr "Не вдалося створити каталог «{}» через те, що «{}»"

#: ../plugins/local.py:121
msgid "'{}' is not a directory"
msgstr "«{}» не є каталогом"

#: ../plugins/local.py:130
msgid "Copying '{}' to local repo"
msgstr "Копіюємо «{}» до локального сховища"

#: ../plugins/local.py:136
msgid "Can't write file '{}'"
msgstr "Не вдалося записати файл «{}»"

#: ../plugins/local.py:151
msgid "Rebuilding local repo"
msgstr "Перезбираємо локальне сховище"

#: ../plugins/leaves.py:32
msgid "List installed packages not required by any other package"
msgstr ""
"Вивести список пакунків, які не потрібні для роботи будь-яких інших пакунків"

#: ../plugins/needs_restarting.py:167
msgid "determine updated binaries that need restarting"
msgstr "визначити оновлені виконувані файли, які потребують перезапуску"

#: ../plugins/needs_restarting.py:172
msgid "only consider this user's processes"
msgstr "брати до уваги лише процеси цього користувача"

#: ../plugins/repoclosure.py:42
msgid "Display a list of unresolved dependencies for repositories"
msgstr "Показати список нерозв’язаних залежностей для сховищ"

#: ../plugins/repoclosure.py:66
msgid "Repoclosure ended with unresolved dependencies."
msgstr "Виконання repoclosure завершилося із нерозв’язаними залежностями."

#: ../plugins/repoclosure.py:139
msgid "check packages of the given archs, can be specified multiple times"
msgstr ""
"перевірити пакунки вказаної архітектури, можна використовувати декілька "
"разів"

#: ../plugins/repoclosure.py:142
msgid "Specify repositories to check"
msgstr "Вказати сховища для перевірки"

#: ../plugins/repoclosure.py:144
msgid "Check only the newest packages in the repos"
msgstr "Перевірити лише найновіші пакунки у сховищах"

#: ../plugins/repoclosure.py:147
msgid "Check closure for this package only"
msgstr "Перевірити замкненість лише для цього пакунка"

#: ../plugins/repomanage.py:44
msgid "Manage a directory of rpm packages"
msgstr "Керування каталогом пакунків rpm"

#: ../plugins/repomanage.py:56
msgid "Pass either --old or --new, not both!"
msgstr "Слід передавати --old або --new, не обидва одразу!"

#: ../plugins/repomanage.py:66
msgid "No files to process"
msgstr "Немає файлів для обробки"

#: ../plugins/repomanage.py:71
msgid "Could not open {}"
msgstr "Не вдалося відкрити {}"

#: ../plugins/repomanage.py:128
msgid "Print the older packages"
msgstr "Вивести старіші пакунки"

#: ../plugins/repomanage.py:130
msgid "Print the newest packages"
msgstr "Вивести найновіші пакунки"

#: ../plugins/repomanage.py:132
msgid "Space separated output, not newline"
msgstr ""
"Відокремлювати записи у виведених даних пробілами, а не символами розриву "
"рядка"

#: ../plugins/repomanage.py:134
msgid "Newest N packages to keep - defaults to 1"
msgstr "Зберігати N найновіших пакунків. Типове значення — 1"

#: ../plugins/repomanage.py:137
msgid "Path to directory"
msgstr "Шлях до каталогу"

#: ../plugins/download.py:41
msgid "Download package to current directory"
msgstr "Отримати пакунок до поточного каталогу"

#: ../plugins/download.py:51
msgid "packages to download"
msgstr "пакунки для отримання"

#: ../plugins/download.py:54
msgid "download the src.rpm instead"
msgstr "отримати замість того src.rpm"

#: ../plugins/download.py:56
msgid "download the -debuginfo package instead"
msgstr "отримати замість цього пакунок -debuginfo"

#: ../plugins/download.py:59
msgid "limit  the  query to packages of given architectures."
msgstr "обмежити пошук пакунків вказаними архітектурами."

#: ../plugins/download.py:61
msgid "resolve and download needed dependencies"
msgstr "визначити і отримати потрібні залежності"

#: ../plugins/download.py:63
msgid ""
"print list of urls where the rpms can be downloaded instead of downloading"
msgstr ""
"вивести список адрес, звідки можна отримати пакунки rpm, замість отримання "
"пакунків"

#: ../plugins/download.py:68
msgid "when running with --url, limit to specific protocols"
msgstr "якщо запущено з --url, обмежитися вказаними протоколами"

#: ../plugins/download.py:105
#, python-format
msgid "Failed to get mirror for package: %s"
msgstr "Не вдалося отримати дзеркало для пакунка: %s"

#: ../plugins/download.py:180
msgid "Exiting due to strict setting."
msgstr "Завершуємо роботу через строгі обмеження."

#: ../plugins/download.py:200
msgid "Error in resolve of packages:"
msgstr ""

#: ../plugins/download.py:217
#, python-format
msgid "No source rpm defined for %s"
msgstr "Не визначено rpm із початковим кодом для %s"

#: ../plugins/download.py:235 ../plugins/download.py:249
#, python-format
msgid "No package %s available."
msgstr "Немає доступного пакунка %s."

#: ../plugins/show_leaves.py:54
msgid "New leaves:"
msgstr "Нові листки:"

#: ../plugins/copr.py:55
msgid "yes"
msgstr "так"

#: ../plugins/copr.py:55
msgid "y"
msgstr "т"

#: ../plugins/copr.py:56
msgid "no"
msgstr "ні"

#: ../plugins/copr.py:56
msgid "n"
msgstr "н"

#: ../plugins/copr.py:71
msgid "Interact with Copr repositories."
msgstr "Працювати зі сховищами Copr."

#: ../plugins/copr.py:72
msgid ""
"\n"
"  enable name/project [chroot]\n"
"  disable name/project\n"
"  remove name/project\n"
"  list --installed/enabled/disabled\n"
"  list --available-by-user=NAME\n"
"  search project\n"
"\n"
"  Examples:\n"
"  copr enable rhscl/perl516 epel-6-x86_64\n"
"  copr enable ignatenkobrain/ocltoys\n"
"  copr disable rhscl/perl516\n"
"  copr remove rhscl/perl516\n"
"  copr list --enabled\n"
"  copr list --available-by-user=ignatenkobrain\n"
"  copr search tests\n"
"    "
msgstr ""
"\n"
"  enable назва/проект [chroot]\n"
"  disable назва/проект\n"
"  remove назва/проект\n"
"  list --installed/enabled/disabled\n"
"  list --available-by-user=ІМ'Я\n"
"  search проект\n"
"\n"
"  Приклади:\n"
"  copr enable rhscl/perl516 epel-6-x86_64\n"
"  copr enable ignatenkobrain/ocltoys\n"
"  copr disable rhscl/perl516\n"
"  copr remove rhscl/perl516\n"
"  copr list --enabled\n"
"  copr list --available-by-user=ignatenkobrain\n"
"  copr search tests\n"
"    "

#: ../plugins/copr.py:98
msgid "List all installed Copr repositories (default)"
msgstr "Список усіх встановлених сховищ Copr (типово)"

#: ../plugins/copr.py:100
msgid "List enabled Copr repositories"
msgstr "Список увімкнених сховищ Copr"

#: ../plugins/copr.py:102
msgid "List disabled Copr repositories"
msgstr "Список вимкнених сховищ Copr"

#: ../plugins/copr.py:104
msgid "List available Copr repositories by user NAME"
msgstr ""
"Список доступних сховищ Copr, які належать користувачу із вказаним ім'ям"

#: ../plugins/copr.py:117
msgid "Warning: we are using non-standard Copr URL '{}'."
msgstr "Попередження: ми використовуємо нестандартну адресу Copr «{}»."

#: ../plugins/copr.py:156 ../plugins/copr.py:177
msgid "Error: "
msgstr "Помилка: "

#: ../plugins/copr.py:157 ../plugins/copr.py:161
msgid "exactly two additional parameters to copr command are required"
msgstr "команді copr слід передавати точно два додаткові параметри"

#: ../plugins/copr.py:178
msgid "use format `copr_username/copr_projectname` to reference copr project"
msgstr ""
"для використання проект copr скористайтеся форматом "
"«copr_користувач/copr_назва_проекту»"

#: ../plugins/copr.py:180
msgid "bad copr project format"
msgstr "помилкове форматування проекту copr"

#: ../plugins/copr.py:186
msgid ""
"\n"
"You are about to enable a Copr repository. Please note that this\n"
"repository is not part of the main distribution, and quality may vary.\n"
"\n"
"The Fedora Project does not exercise any power over the contents of\n"
"this repository beyond the rules outlined in the Copr FAQ at\n"
"<https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr>,\n"
"and packages are not held to any quality or security level.\n"
"\n"
"Please do not file bug reports about these packages in Fedora\n"
"Bugzilla. In case of problems, contact the owner of this repository.\n"
"\n"
"Do you want to continue?"
msgstr ""
"\n"
"Ви наказали системі увімкнути сховище Copr. Будь ласка, зауважте,\n"
"що це сховище не є частиною основного дистрибутива, тому може\n"
"містити пакунки неналежної якості.\n"
"\n"
"Проект Fedora ніяким чином не впливає на вміст цього\n"
"сховища, окрім правил, які визначено у списку питань щодо Copr тут:\n"
"<https://docs.pagure.org/copr.copr/user_documentation.html#what-i-can-build-in-copr>,\n"
"а пакунки не проходять жодних рівнів забезпечення якості чи захищеності.\n"
"\n"
"Будь ласка, не повідомляйте про виявлені у цих пакунках вади до\n"
"системи стеження за вадами Fedora. Якщо виникнуть якісь проблеми,\n"
"зв’яжіться із власником цього сховища пакунків.\n"
"\n"
"Ви справді хочете продовжити виконання цієї дії?"

#: ../plugins/copr.py:201
msgid "Repository successfully enabled."
msgstr "Сховище успішно увімкнено."

#: ../plugins/copr.py:205
msgid "Repository successfully disabled."
msgstr "Сховище успішно вимкнено."

#: ../plugins/copr.py:209
msgid "Repository successfully removed."
msgstr "Сховище успішно вилучено."

#: ../plugins/copr.py:213 ../plugins/copr.py:476
msgid "Unknown subcommand {}."
msgstr "Невідома підкоманда {}."

#: ../plugins/copr.py:242
msgid "Can't parse repositories for username '{}'."
msgstr "Не вдалося обробити сховища для користувача '{}'."

#: ../plugins/copr.py:245
msgid "List of {} coprs"
msgstr "Список copr {}"

#: ../plugins/copr.py:253
msgid "No description given"
msgstr "Опис не надано"

#: ../plugins/copr.py:265
msgid "Can't parse search for '{}'."
msgstr "Не вдалося обробити пошук для '{}'."

#: ../plugins/copr.py:268
msgid "Matched: {}"
msgstr "Відповідник: {}"

#: ../plugins/copr.py:276
msgid "No description given."
msgstr "Опис не надано."

#: ../plugins/copr.py:289
msgid "Safe and good answer. Exiting."
msgstr "Безпечна і добра відповідь. Завершуємо роботу."

#: ../plugins/copr.py:296
msgid "This command has to be run under the root user."
msgstr "Команду слід віддавати від імені користувача root."

#: ../plugins/copr.py:356
msgid ""
"This repository does not have any builds yet so you cannot enable it now."
msgstr ""
"У цьому сховищі ще немає нічого зібраного, отже ви не можете його зараз "
"увімкнути."

#: ../plugins/copr.py:359
msgid "Such repository does not exist."
msgstr "Такого сховища не існує."

#: ../plugins/copr.py:380
msgid "Failed to disable copr repo {}/{}"
msgstr "Не вдалося вилучити сховище copr {}/{}"

#: ../plugins/copr.py:393 ../plugins/copr.py:431
msgid "Unknown response from server."
msgstr "Невідома відповідь від сервера."

#: ../plugins/copr.py:415
msgid "Interact with Playground repository."
msgstr "Працювати зі сховищем Playground."

#: ../plugins/copr.py:420
msgid ""
"\n"
"You are about to enable a Playground repository.\n"
"\n"
"Do you want to continue?"
msgstr ""
"\n"
"Ви наказали системі увімкнути сховище Playground.\n"
"\n"
"Ви справді хочете продовжити виконання цієї дії?"

#: ../plugins/copr.py:466
msgid "Playground repositories successfully enabled."
msgstr "Сховища playground успішно увімкнено."

#: ../plugins/copr.py:469
msgid "Playground repositories successfully disabled."
msgstr "Сховища playground успішно вимкнено."

#: ../plugins/copr.py:473
msgid "Playground repositories successfully updated."
msgstr "Сховища playground успішно оновлено."

#: ../plugins/debug.py:53
msgid "dump information about installed rpm packages to file"
msgstr "створити дамп даних щодо встановлених пакунків rpm у файлі"

#: ../plugins/debug.py:67
msgid "do not attempt to dump the repository contents."
msgstr "не намагатися створити дамп вмісту сховища."

#: ../plugins/debug.py:70
msgid "optional name of dump file"
msgstr "необов’язкова назва файла дампу"

#: ../plugins/debug.py:95
#, python-format
msgid "Output written to: %s"
msgstr "Результат записано до %s"

#: ../plugins/debug.py:172
msgid "restore packages recorded in debug-dump file"
msgstr "відновити пакунки, записані до файла debug-dump"

#: ../plugins/debug.py:183
msgid "output commands that would be run to stdout."
msgstr "вивести команди, які буде віддано, до stdout."

#: ../plugins/debug.py:186
msgid "Install the latest version of recorded packages."
msgstr "Встановити найсвіжішу версію записаних пакунків."

#: ../plugins/debug.py:189
msgid ""
"Ignore architecture and install missing packages matching the name, epoch, "
"version and release."
msgstr ""
"Ігнорувати архітектуру і встановити усі пакунки, яких не вистачає, за "
"відповідністю назви, епохи, версії та випуску."

#: ../plugins/debug.py:194
msgid "limit to specified type"
msgstr "обмежитися вказаним типом"

#: ../plugins/debug.py:196
msgid "name of dump file"
msgstr "назва файла дампу"

#: ../plugins/debug.py:264
#, python-format
msgid "Package %s is not available"
msgstr "Пакунок %s недоступний"

#: ../plugins/debug.py:274
#, python-format
msgid "Bad dnf debug file: %s"
msgstr "Помилковий файл діагностики dnf: %s"
